response_operations_ui/views/reporting_units.py

```diff
@@ -169,7 +169,11 @@ def search_reporting_units():
 def view_resend_verification(ru_ref, party_id):
     logger.debug("Re-send verification email requested", ru_ref=ru_ref, party_id=party_id)
     respondent = party_controller.get_respondent_by_party_id(party_id)
-    return render_template('re-send-verification-email.html', ru_ref=ru_ref, email=respondent['emailAddress'])
+    if 'pendingEmailAddress' in respondent:
+        return render_template('re-send-verification-email.html', ru_ref=ru_ref,
```

Comment:
```
email = respondent['pendingEmailAddress'] if 'pendingEmailAddress' in respondent else respondent['emailAddress']

return render_template('re-send-verification-email.html', ru_ref=ru_ref, email=email)
```

I feel like its easier to read this way ^^

Reply:
Also looks like you will always have a key 'pendingEmailAddress' in respondent. [source](https://github.com/ONSdigital/ras-party/blob/4e4b293c3f6e555998e174d50ed566f901985ef0/ras_party/models/models.py#L266)

Maybe 
`email = respondent['pendingEmailAddress'] if respondent['pendingEmailAddress'] else respondent['emailAddress'] ` ??

Reply:
Done 👍